src/main/java/models/AccountModel.java

```diff
@@ -0,0 +1,33 @@
+package models;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+public class AccountModel {
+    public CustomerModel customerModel;
+    public String loginName;
+    public String password;
+
+    @Builder.Default
+    public Boolean isSubscribedToNewsletter = false;
+
+    public static AccountModel withCustomerModel(CustomerModel customerModel) {
+        return AccountModel.builder()
+                .customerModel(customerModel)
+                .loginName(CustomerModelUtils.generateLoginName())
```

Comment:
if password and login is no longer connected to customer model I think we should move those generate methods here:)

Reply:
Right. I forgot about them, thanks
```java
    private static String generateLoginName() {
        return CustomerModelUtils.fakerEN.name().lastName();
    }

    private static String generatePassword() {
        return CustomerModelUtils.fakerEN.internet().password();
    }
 ```